package.json

```diff
@@ -6,6 +6,7 @@
     "@testing-library/jest-dom": "^5.15.0",
     "@testing-library/react": "^11.2.7",
     "@testing-library/user-event": "^12.8.3",
+    "big.js": "^6.1.1",
```

Comment:
- [ ] In `.eslintrc.json` you need to use the exact same rules listed [here](https://github.com/microverseinc/linters-config/blob/master/react-redux/.eslintrc.json).